astropy/visualization/wcsaxes/tests/test_misc.py

```diff
@@ -503,7 +500,7 @@ def test_set_labels_with_coords(ignore_matplotlibrc, frame_class):
         assert ax.coords[i].get_axislabel() == labels[i]
 
 
-@pytest.mark.skipif('not MATPLOTLIB_33')
+@pytest.mark.skipif('not MATPLOTLIB_EQ_33')
```

Comment:
Not sure why this test only runs for 3.3. Seems awfully specific.

Reply:
Just tested this on master with matplotlib 3.0.3, where it is not skipped and passes, and I think there is no reason to assume it won't anymore in 3.4+, so I think the whole decorator can be removed now.
```suggestion
```

Reply:
@Cadair added this recently, it was quite specific because the bounding box calculation changes a little from version to version.

Reply:
So it's just passing with 3.0.3 by chance? I tried with `np.allclose(..., rtol=1.e-9, atol=1.e-12)` and it still succeeds.

Reply:
Conversely, what it the purpose of a test for a single minor version, even if it is the current one right now?

Reply:
The bounding box calculation is dependent on mpl versions, which is why this decorator was added in https://github.com/astropy/astropy/pull/10797

Reply:
OK, not sure why I could not reproduce it then; the changes may be related to the treatment of the 
`ignore_label`/`for_layout_only` option, which was only introduced post 3.0.
https://github.com/matplotlib/matplotlib/blob/868504e3ffde5b14cecb4bf2da27af90a0a60d91/lib/matplotlib/axis.py#L1105
Then there is probably no point in trying to find all versions where it works or if or how it might change again with 3.4...

Reply:
As an aside, that test did run for #10797 in mpl302, mpl310 and mpldev (because of the mistaken `MATPLOTLIB_33` definition) and evidently passed all...

Reply:
I added a comment to explain then. Thanks!

Reply:
As an aside, that test did run for #10797 in mpl302, mpl310 and mpldev (because of the mistaken `MATPLOTLIB_33` definition) and evidently passed all.

Reply:
Well, actually the tests (all 3.x ones) succeeded once the target `x1` was changed in https://github.com/dstansby/astropy/commit/90366e, not when the versions were (not) restricted, so I'd take the "very version-dependent" with a grain of salt.

Reply:
Or rather, with the original `ax.bbox_x1=576`, `ax.bbox_y1=432` the mpl2x tests only were failing; after changing `x1` to 579.5 mpl2 now passed that but still failed on `y1`, while mpl3x then began to fail on the new `x1`.
So it seems safe to say all 2.x versions create `ax.bbox_x1=579.5`, `ax.bbox_y1=435.5` and all 3.x versions `ax.bbox_x1=576`, `ax.bbox_y1=432`:
https://app.circleci.com/pipelines/github/astropy/astropy?branch=pull%2F10797